pycobertura/reporters.py

```diff
@@ -14,77 +14,54 @@
 file_row = namedtuple("FileRow", row_attributes)
 file_row_missed = namedtuple("FileRowMissed", row_attributes + ["missed_lines"])
 
+headers_without_missing = ["Filename", "Stmts", "Miss", "Cover"]
+headers_missing = ["Filename", "Stmts", "Miss", "Cover", "Missing"]
```

Comment:
To be more symmetric with its counterpart `headers_without_missing`, would it make sense to rename `headers_missing` as such?

```suggestion
headers_with_missing = ["Filename", "Stmts", "Miss", "Cover", "Missing"]
```

Reply:
Done